test/e2e/__tests__/code-actions.ts

```diff
@@ -30,25 +32,67 @@ describe('Code actions', () => {
 
 		await waitForDiagnostics(editor);
 
-		const actions = await getCodeActions(editor, new Selection(1, 11, 1, 11));
+		const actions = await getCodeActions(editor, new Selection(3, 11, 3, 11));
 
 		assert.equal(actions.length, 3);
 		assertTextEdits(actions[0].edit?.get(editor.document.uri), [
 			{
-				newText: `  /* stylelint-disable-next-line value-keyword-case */${EOL}`,
-				range: [1, 0, 1, 0],
+				newText: `  /* stylelint-disable-next-line color-no-invalid-hex */${EOL}`,
```

Comment:
In the future, if value-keyword-case corresponds to EditInfo, the test will break, so we have changed it to a test that uses a different rule